package.json

```diff
@@ -46,7 +46,7 @@
   ],
   "typings": "dist/packages/index",
   "dependencies": {
-    "@emotion/css": "11.11.0",
+    "@emotion/css": "11.11.2",
```

Comment:
Why not start switchting to version ranges? :)

Reply:
That's the next piece for me. I'd rather do this first and it's less messy so I can isolate it to only version ranges and not upgrades with that too.